extensions/amp-analytics/0.1/linker-manager.js

```diff
@@ -244,9 +244,12 @@ export class LinkerManager {
 
     // If no domains given, default to friendly domain matching.
     if (!domains) {
-      const {sourceUrl} = Services.documentInfoForDoc(this.ampdoc_);
+      const {sourceUrl, canonicalUrl} =
+          Services.documentInfoForDoc(this.ampdoc_);
       const sourceOrigin = urlService.parse(sourceUrl).hostname;
-      if (!isFriendlyDomains(sourceOrigin, hostname)) {
+      const canonicalOrigin = urlService.parse(canonicalUrl).hostname;
+      if (!areFriendlyDomains(sourceOrigin, hostname)
+          && !areFriendlyDomains(canonicalOrigin, hostname)) {
```

Comment:
sorry. I would have hoped for `||`. This way we get the broadest coverage. It also feels right given that both the domains are owned by the pub.

Reply:
broader coverage comes with `&&` . I have test covers that